locales/en.default.json

```diff
@@ -292,8 +281,10 @@
             "company": "Company",
             "request_shipping_quote": "Request a shipping quote",
             "submit_quote": "Submit quote request",
-            "update_quote": "Update quote",
             "remove": "Remove",
+            "product_added_request": "Product is added to quote request",
```

Comment:
you copied this, not moved. remove `product_added_request`, `go_request`, `update_request` from `cart.general`